src/services/gateways/aws.ts

```diff
@@ -170,6 +172,8 @@ export class AWS {
     this.ecsClient = new ECSClient(config);
     this.rdsClient = new RDSClient(config);
     this.cwClient = new CloudWatchLogsClient(config);
+    // Service endpoint need to be in the same region where we have the s3 bucket with the lambda function and where the user runs the cloudformation template
+    this.cfnClient = new CloudFormationClient({credentials: config.credentials, region: 'us-east-2'});
```

Comment:
do we need to set the region? I input a region into the ui, the stack has `us-east-2` hardcoded and finally I can toggle across regions in the AWS ui to create the stack in some other region? can we just let the user pick the `region` they want to manage based on where the stack gets created?

Reply:
I can double check but the issue here is that the stack need to be created in the same region where **our** s3 bucket with the lambda function will live, otherwise the CloudFormation stack will fail on creation. That's why I've created this special client with a fixed region (the region where the s3 bucket live right now)

Reply:
I see, we would have to create this dynamically in the ux flow I am describing